sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityValue.java

```diff
@@ -6,84 +6,148 @@
  */
 package com.powsybl.sensitivity;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.powsybl.commons.PowsyblException;
 
+import java.io.IOException;
+import java.io.UncheckedIOException;
 import java.util.Objects;
 
 /**
- * Elementary result value of sensitivity analysis.
- * Associates a value to a sensitivity factor.
+ * Elementary result value of a sensitivity analysis, given the sensitivity factor and a contingency id (use null to get
+ * a pre-contingency value). The value is the impact of the variable change on the monitored equipment. The function
+ * reference gives the level of the function in the network pre-contingency state.
```

Comment:
Isn't the function reference the level of the function in the network in the contingency state defined by the contingency context of the factor?